Code-4-Community/hands-across-the-sea-frontend
# Style Select School Container

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/XXXXXXXX/pulses/XXXXXXX)

Fixes select school container to match the designs.

Also adds a FormButtons component which makes it easy to render submit and other navigation buttons to a form.

## Screenshots
![select-school](https://user-images.githubusercontent.com/55663618/118534090-d7a4d700-b716-11eb-96aa-bff2da0dc716.png)

## Verification Steps

Make sure form still works as expected. Submit button should be disabled before a school is selected.
